lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
```

Comment:
Please remove this change from this branch.  Try to restrict each branch/PR to fixing/adding one issue/feature.